catroid/src/org/catrobat/catroid/content/bricks/WhenRaspiPinChangedBrick.java

```diff
@@ -55,6 +55,10 @@ public WhenRaspiPinChangedBrick(RaspiInterruptScript script) {
 		if (script != null) {
 			pinString = script.getPin();
 			eventString = script.getEventValue();
+
+			if (script.isCommentedOut()) {
+				setCommentedOut(true);
+			}
 		}
```

Comment:
Why is the clone method not altered in order to set the cloned brick comment state ?